webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -64,6 +64,10 @@
           </extension-registry-components>
         </v-card-text>
       </v-card>
+      <extension-registry-components
+        :params="dropFilesParams"
+        name="ActivityComposerDropZone"
+        type="activity-composer-drop-zone" />
```

Comment:
No need to use an `extension-registry-components`, it will be all time one single component for drag and drop management. I think it would be better to change this to a `Vue directive`

```suggestion
```

Reply:
done